cnv/cnv_users_guide/cnv-cloning-vm-disk-into-new-datavolume-block.adoc

```diff
@@ -0,0 +1,21 @@
+[id="cnv-cloning-vm-disk-into-new-datavolume-block"]
+= Cloning a virtual machine disk into a new block storage DataVolume
+include::modules/cnv-document-attributes.adoc[]
+:context: cnv-cloning-vm-disk-into-new-datavolume-block
+toc::[]
+
+You can clone the PersistentVolumeClaim (PVC) of a virtual machine disk into
+a new block DataVolume by referencing the source PVC in your DataVolume configuration
```

Comment:
You can clone the PersistentVolumeClaim (PVC) of a virtual machine disk into
a new block DataVolume by referencing the source PVC in your DataVolume manifest

Reply:
CNV docs uses 'configuration' for all objects. OpenShift docs use 'manifest' to refer to Operator and Install manifests.  To stay consistent, I won't change it this close to the release.
But if we're using incorrect terminology we should address that on a larger scale. Do you mind elaborating on why these should be 'manifest'?